src/main/java/hudson/plugins/jira/updater/ExplicitUpdaterIssueSelector.java

```diff
@@ -0,0 +1,37 @@
+package hudson.plugins.jira.updater;
```

Comment:
Move to hudson.plugins.jira.selector, along with the default one. Rename to AbstractIssueSelector.